resource-managers/kubernetes/integration-tests/dev/dev-run-integration-tests.sh

```diff
@@ -180,4 +190,4 @@ properties+=(
   -Dlog4j.logger.org.apache.spark=DEBUG
 )
 
-$TEST_ROOT_DIR/build/mvn install -f $TEST_ROOT_DIR/pom.xml -pl resource-managers/kubernetes/integration-tests $BUILD_DEPENDENCIES_MVN_FLAG -Pscala-$SCALA_VERSION -P$HADOOP_PROFILE -Pkubernetes -Pkubernetes-integration-tests ${properties[@]}
+(cd $TEST_ROOT_DIR; ./build/mvn install -pl resource-managers/kubernetes/integration-tests $BUILD_DEPENDENCIES_MVN_FLAG -Pscala-$SCALA_VERSION -P$HADOOP_PROFILE -Pkubernetes -Pkubernetes-integration-tests ${properties[@]})
```

Comment:
I noticed that all the examples of `dev-run-integration-tests.sh` in `integration-tests/README.md` don't work because  `scala-style-config.xml` is not found in `integration-tests`.
This change is to resolve the issue.

Reply:
Does this work? I'm testing your PR and hit the following.
```
$ cd resource-managers/kubernetes/integration-tests
$ ./dev/dev-run-integration-tests.sh --docker-file ../docker/src/main/dockerfiles/spark/Dockerfile.java17
++ git rev-parse --show-toplevel
+ TEST_ROOT_DIR=/Users/dongjoon/APACHE/spark-merge
+ DEPLOY_MODE=minikube
+ IMAGE_REPO=docker.io/kubespark
+ SPARK_TGZ=N/A
+ IMAGE_TAG=N/A
+ JAVA_IMAGE_TAG=
+ BASE_IMAGE_NAME=
+ JVM_IMAGE_NAME=
+ PYTHON_IMAGE_NAME=
+ R_IMAGE_NAME=
+ DOCKER_FILE=
+ SPARK_MASTER=
+ NAMESPACE=
+ SERVICE_ACCOUNT=
+ CONTEXT=
+ INCLUDE_TAGS=k8s
+ EXCLUDE_TAGS=
+ JAVA_VERSION=8
+ BUILD_DEPENDENCIES_MVN_FLAG=-am
+ HADOOP_PROFILE=hadoop-3.2
+ MVN=/Users/dongjoon/APACHE/spark-merge/build/mvn
++ /Users/dongjoon/APACHE/spark-merge/build/mvn help:evaluate -Dexpression=scala.binary.version
++ grep -v INFO
++ grep -v WARNING
++ tail -n 1
+ SCALA_VERSION=2.12
+ export SCALA_VERSION
+ echo 2.12
2.12
+ ((  2  ))
+ case $1 in
+ DOCKER_FILE=../docker/src/main/dockerfiles/spark/Dockerfile.java17
+ shift
+ shift
+ ((  0  ))
+ properties=(-Djava.version=$JAVA_VERSION -Dspark.kubernetes.test.sparkTgz=$SPARK_TGZ -Dspark.kubernetes.test.imageTag=$IMAGE_TAG -Dspark.kubernetes.test.imageRepo=$IMAGE_REPO -Dspark.kubernetes.test.deployMode=$DEPLOY_MODE -Dtest.include.tags=$INCLUDE_TAGS)
+ '[' -n '' ']'
+ '[' -n ../docker/src/main/dockerfiles/spark/Dockerfile.java17 ']'
+ properties=(${properties[@]} -Dspark.kubernetes.test.dockerFile=$(realpath $DOCKER_FILE))
++ realpath ../docker/src/main/dockerfiles/spark/Dockerfile.java17
./dev/dev-run-integration-tests.sh: line 153: realpath: command not found
```

Reply:
Hmm, `realpath` seems not to be a standard command. I tested on Pop!_OS 20.04.
I'll look for another way.

Reply:
Python's `os.path.realpath` should work on both Linux and macOS.

Reply:
No~ It's Spark function in `utils.sh`.

Reply:
Oh..

Reply:
Ah, Linux has `realpath` built-in command (coreutils) but we have own `realpath`.